pkg/tracking/store/sql/experiments.go

```diff
@@ -78,6 +78,17 @@ func (s TrackingSQLStore) CreateExperiment(input *protos.CreateExperiment) (stri
 	return strconv.Itoa(int(*experiment.ID)), nil
 }
 
+func (s TrackingSQLStore) UpdateExperiment(experiment *protos.Experiment) *contract.Error {
```

Comment:
@jgiannuzzi the Python code went with `rename_experiment`. Do we want to do the same here?

Reply:
yes, I think we should keep the Go code _naming_ identical to the Python code - if we strongly feel that the name is badly chosen or inconsistent, we should probably raise a PR upstream but keep in sync in the meantime